docs/tutorial/1-serialization.md

```diff
@@ -45,7 +45,7 @@ We'll need to add our new `snippets` app and the `rest_framework` app to `INSTAL
     INSTALLED_APPS = (
         ...
         'rest_framework',
-        'snippets',
+        'snippets.apps.SnippetsConfig',
```

Comment:
I can't understand that change in Django - how come the built-in apps don't need that style? What about `rest_framework`?